indexer-lib/src/periodic_processor.rs

```diff
@@ -120,7 +127,13 @@ impl PeriodicProcessor {
                     self.handle_sender_resolution(r)?;
                 }
                 Notification::Tick => {
-                    self.tick_work()?;
+                    let tick_result = self.tick_work();
+
+                    if let Err(error) = tick_result {
+                        error!("{:?}", error);
+                        panic!();
+                    }
+
                     self.job_done_tx.send_blocking(())?;
                 }
```

Comment:
need validation

Reply:
fixed in [4eb236f62fd863a7a5bf8fab190e63290ab740f2](https://github.com/K-Kluster/kasia-indexer/pull/10/commits/4eb236f62fd863a7a5bf8fab190e63290ab740f2)

please review because i'm not used to that pattern:
```rs
Notification::Tick => {
    let _ = self.tick_work().inspect_err(|error| {
        error!("{:?}", error);
    });

    self.job_done_tx.send_blocking(())?;
}
```